src/main/kotlin/com/atlassian/performance/tools/report/Util.kt

```diff
@@ -0,0 +1,18 @@
+package com.atlassian.performance.tools.report
+
+import com.atlassian.performance.tools.report.api.result.EdibleResult
+
+internal class Util {
+
+    companion object {
+        fun toDoubleArray(
+            result: EdibleResult,
+            label: String
+        ): DoubleArray {
+            return result.actionMetrics
+                .filter { it.label == label }
+                .map { it.duration.toMillis() / 1000.0 }
```

Comment:
Why are we dividing by a thousand?
Do we want to convert to seconds? If so, then: `it.duration.seconds.toDouble()`. Also - why convert to seconds?

Reply:
Interesting, when I applied my suggestion, the unit tests started failing. It seems that we'd lose precision this way 😬 
Good call on the unit tests 🙂

Reply:
Using Nanos now. Default from `ActionMetricsReader`